espaloma/mm/implicit.py

```diff
@@ -95,32 +67,22 @@ def _gbsa_obc2_energy_omm(
             psi3_coefficient * psi ** 3)
 
     B = 1 / (1 / offset_radius - torch.tanh(psi_term) / radii)
-    assert (B.shape == (N,))
 
     E = 0.0
     # single particle
     # ACE
-    ACE_individual_terms = surface_tension * (radii + probe_radius) ** 2 * (
-                radii / B) ** 6
-    ACE = torch.sum(ACE_individual_terms)
-    assert (ACE_individual_terms.shape == (N,))
-    E += ACE
+    E += torch.sum(
+        surface_tension * (radii + probe_radius) ** 2 * (radii / B) ** 6)
 
     # on-diagonal
-    on_diagonal = -0.5 * (
-            1 / solute_dielectric - 1 / solvent_dielectric) * charges ** 2 / B
-    assert (on_diagonal.shape == (N,))
-    E += torch.sum(on_diagonal)
+    E += torch.sum(-0.5 * (
+            1 / solute_dielectric - 1 / solvent_dielectric) * charges ** 2 / B)
 
     # particle pair
     # note: np.outer --> torch.ger
-    assert (B.ndim == 1)
-
     f = torch.sqrt(r ** 2 + torch.ger(B, B) * torch.exp(
         -r ** 2 / (4 * torch.ger(B, B))))
     charge_products = torch.ger(charges, charges)
-    assert (f.shape == (N, N))
-    assert (charge_products.shape == (N, N))
 
     ixns = - (
```

Comment:
Isn't this missing a -138.935485 conversion from nm/(proton_charge**2) to kJ/mol? The docstring says "everything is in OpenMM native units".

Reply:
Comparing with https://github.com/openmm/openmm/blob/master/wrappers/python/simtk/openmm/app/internal/customgbforces.py#L381-L382

Reply:
It looks like you might pre-multiply the charges by `sqrt(138.935485)`? If so, you should probably document that in the docstring.

Reply:
Gahh -- you're right -- I had dropped this in the current conversion! Thank you for catching this. Charges are not assumed to be premultiplied by `sqrt(138.935485)` , will clarify docstring...

(This conversion was present but poorly labeled in the numpy/jax implementation in bayes-implicit-solvent.)